src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance;
+
+    private HibernateUtil() {
+        instance = new Configuration().configure().buildSessionFactory();
```

Comment:
The constructor should not be responsible for initializing the `SessionFactory`. Consider using a static block or a synchronized method for initialization to ensure thread safety and proper singleton behavior.